docs/examples/spatial_transformation/histogram_matching.jl

```diff
@@ -0,0 +1,77 @@
+# ---
+# cover: assets/chelsea.png
+# title: Histogram Matching
+# author: Ashwani Rathee
+# date: 2020-11-21
+# ---
+
+# This demo demonstrates the feature of histogram matching.
+
+# In histogram matching,the pixels of source image is manipulated so that the histogram 
+# of reference image matches with source image's histogram.
```

Comment:
```suggestion
During histogram matching, the pixels of the source image is manipulated so that the histogram of it matches with that of the reference image.
```